### DISCUSSION TITLE
Please tell me, is it possible to add all the outstanding todos to the daily template?

### DISCUSSION THREAD

**Discussion by q7w7e7r** [TriliumNext/Trilium#6021] [Ideas] [answered]
For example, today is December 7th, and the task was on December 6th, it is displayed in the notes catalog for the 6th day, but it is not available for the 7th day.

**Comment by CobriMediaJulien** [comment]
I don´t think you can do this right now. Maybe this helps?

https://github.com/zadam/trilium/discussions/2946

there is also a kanban plugin.

**Reply by q7w7e7r** [reply]
Thanks, I tried it, but for some reason it refuses to work for me. I'm probably doing something wrong.

**Comment by Nriver** [comment, marked as answer]
You can try this feature of trilium-py https://github.com/Nriver/trilium-py?tab=readme-ov-file#move-yesterdays-unfinished-todo-to-today.

> As the title suggests, you can move yesterday's unfinished things to today. Unfinished todo's will be deleted from yesterday's note.

**Reply by q7w7e7r** [reply]
Thanks, it looks like the truth. It remains to figure out where to implement this command.
